framework/src/st_things/things_stack/framework/things_req_handler.c

```diff
@@ -190,6 +190,84 @@ OCEntityHandlerResult send_response(OCRequestHandle request_handle, OCResourceHa
 	return eh_result;
 }
 
+static OCEntityHandlerResult convert_ap_infor(things_resource_s *target_resource, access_point_info_s *p_list, int list_cnt)
+{
+	things_representation_s *rep = NULL;
+
+	if (list_cnt == 0 || p_list == NULL) {
```

Comment:
@JeongJunSik  this target_resource is calling through internal Api.  In internal APi target_resource was already check of null condition.So that here no need to check again.
OCEntityHandlerResult handle_message(things_resource_s *target_resource)
{
	OCEntityHandlerResult eh_result = OC_EH_ERROR;

	if (NULL == target_resource) {
		THINGS_LOG_D(TAG, "Request Item is NULL.");
		return OC_EH_ERROR;
	}